src/app/message/send/[userId]/[variety]/completion/_components/submitButton.tsx

```
@@ -0,0 +1,57 @@
+
+  const { mutateAsync, isPending } = usePostSendMessage()
+
+  const onSubmit = handleSubmit(async (formValues) => {
+    try {
+      const response = await mutateAsync({
+        receiverId: Number(loginId),
+        emotionId: emotionId || 1,
+        content: formValues.message,
+      })
+
+      if (response && response.dmId) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)